src/synthesize.jl

```diff
@@ -5,27 +5,29 @@ import ..ContinuumOpacity
     synthesize(atm, linelist, λs, [metallicity, [alpha]]; abundances=Dict())
 
 Solve the transfer equation in the model atmosphere `atm` with the transitions in `linelist` at the 
-wavelengths `λs` to get the resultant astrophysical flux at each wavelength.
+wavelengths `λs` [Å] to get the resultant astrophysical flux at each wavelength.
 
 Other arguments:
 - `metallicity`, i.e. [metals/H] is log_10 solar relative
 - `vmic` (default: 0) is the microturbulent velocity, ξ, in km/s.
 - `abundances` are A(X) format, i.e. A(x) = log_10(n_X/n_H), where n_X is the number density of X.
-- `line_window` (default: 10): the farthest any line can be from the provede wavelenth range 
-   before it is discarded.
-
+- `line_window` (default: 10): the farthest any line can be from the provide wavelenth range range
+   before it is discarded (in Å).
 Uses solar abundances scaled by `metallicity` and for those not provided.
 """
 function synthesize(atm, linelist, λs::AbstractVector{F}, metallicity::F=0.0; vmic=1.0, 
                     abundances=Dict(), line_window::F=10.0) where F <: AbstractFloat
+    #work in cm
+    λs *= 1e-8
```

Comment:
Would you mind explaining your rational as to why the arguments are specified in Å instead of cm? (I'm just curious)

Additionally I think you should be careful about modifying `λs` in-place as it produces an unexpected side-effect. I think you should either document in the docstring that this is one of the consequences of the function call or you should store the values in a new array.

Reply:
Even though I think it's easier if the code internals are all cgs, I think the user-facing/top-level function should take Angstroms since that's the standard in the field.

> Additionally I think you should be careful about modifying λs in-place as it produces an unexpected side-effect. I think you should either document in the docstring that this is one of the consequences of the function call or you should store the values in a new array.

Good catch.  I will address this.